README.md

```diff
@@ -71,7 +71,13 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Not deployed
+> <a href="https://shayan1234554321.github.io/warrior-cars/">Live</a>
```

Comment:
- Great Job for adding the live demo link,👍👍 but it is not the paged as needed please check it and fix it to render the as expected for end users 👍👍

Reply:
Well the branch is not merged , thats why its not the same yet. After approval, i will merge the branch and will be the same after that @binhussen

Reply:
- Great job for deploying the `main` branch,👍👍 however it is needed to deploy the feature branch which is `Styling-home-for-desktop-mobile-version` because the reviewer will check only he feature branch, not your main branch👍👍